spec/scenic/adapters/postgres/views_spec.rb

```diff
@@ -0,0 +1,23 @@
+require "spec_helper"
+
+module Scenic
+  module Adapters
+    describe Postgres::Views, :db do
+      it "does not query for materialized views if not supported" do
+        connection = ActiveRecord::Base.connection
+        allow(connection).to receive(:supports_materialized_views?)
+          .and_return(false)
+        ActiveRecord::Base.connection.execute(
+          "CREATE VIEW greetings AS SELECT text 'hi' AS greeting",
+        )
+        ActiveRecord::Base.connection.execute(
+          "CREATE MATERIALIZED VIEW farewells AS SELECT text 'bye' AS text",
+        )
```

Comment:
Yeah. We've been chatting about how we would even test reliably against multiple versions of postgres (no CI services support this very well). We'll cross that bridge at some point.